cmd/sealer/cmd/image/push.go

```diff
@@ -34,18 +38,29 @@ var exampleForPushCmd = `
 func NewPushCmd() *cobra.Command {
 	pushCmd := &cobra.Command{
 		Use:   "push",
-		Short: "push ClusterImage to remote registry",
+		Short: "push sealer image to remote registry",
```

Comment:
good code describes . so , could we walkthrough all sealer cmd description to replace `ClusterImage` to `sealer image`.